src/v/cluster/members_manager.cc

```diff
@@ -314,13 +314,18 @@ members_manager::handle_join_request(model::broker broker) {
         // we do not use revisions in raft0 configuration, it is always revision
         // 0 which is perfectly fine. this will work like revision less raft
         // protocol.
+        model::node_id id = broker.id();
         return _raft0
           ->add_group_members({std::move(broker)}, model::revision_id(0))
-          .then([](std::error_code ec) {
+          .then([id](std::error_code ec) {
               if (!ec) {
                   return ret_t(join_reply{true});
               }
-
+              vlog(
```

Comment:
why not copy the broker here too

and log the broker that was attempted

Reply:
I thought that node_id is more readable, can make it broker